bind.go

```diff
@@ -42,7 +42,11 @@ func (b *DefaultBinder) Bind(i interface{}, c Context) (err error) {
 	ctype := req.Header.Get(HeaderContentType)
 	switch {
 	case strings.HasPrefix(ctype, MIMEApplicationJSON):
-		if err = json.NewDecoder(req.Body).Decode(i); err != nil {
+		d := json.NewDecoder(req.Body)
+		if strict {
+			d.DisallowUnknownFields()
```

Comment:
I think it would be more helpful to rename "strict mode" to `DisallowUnknownFields`.  I had to look at the source to figure out what this does.

Also, maybe it would be possible to allow passing a custom `json.Decoder`?

Reply:
Yes, you're probably right. This will also give us compatability with go != 1.10. I'll do new PR then.